### ISSUE TITLE
When will the bot be back :(

### ISSUE THREAD

**Issue by iKonatai** [Novalys/PokemonGo-Bot-SimpleGUI#278]

**Comment by Sbrown-WilliamsCrusher** [comment]
There is no eta. We will announce on issue #271 and on the discord channel.

Please close this :smile: as it is not an issue

**Comment by iKonatai** [comment]
@Sbrown-WilliamsCrusher Where is the discord channel and how do I find it? I wish to join, I was just asking as Im not good with this website

**Comment by Sbrown-WilliamsCrusher** [comment]
https://discord.gg/hEdUFqv

**Comment by Zathoros** [comment]
According to the official Reddit thread of the people working on fixing the API, it has now been fixed. Just need to wait on the new API to be added to this bot.

**Comment by exsypher** [comment]
@Zathoros Correct they hacked it finally after working 3 days without break. But there are still some issues which need some time to be fixed before ppl can use it
1. > This API is not flag-proof. Any account using this API will easily be flagged as not playing through the official app. For now the devs have had enough of it and you can't blame them.
2. > ...all API requests will appear to Niantic to be coming from IOS users, this is wierd if it is matched with a device which normally runs Android

**Comment by yupdon** [comment]
@exsypher Even so I catch pokemon for my wife all the time, from my android, while she has an Iphone.... They can suck it.

**Comment by exsypher** [comment]
@yupdon well you can do what you if you don't care (like me) about getting perma banned

**Comment by yupdon** [comment]
@exsypher Well it's just unlikely to get banned for that...that's all

**Comment by flyzow** [comment]
So when can we DL it working ? x)

**Comment by exsypher** [comment]
@flyzow depends on how long they need to fully finish the api and the devs from this bot need to implement the new api in their code...

**Comment by flyzow** [comment]
@exsypher  oh ok ok, thanks =)

**Comment by yupdon** [comment]
@Everyone, necro bot said their up if that's the case I'll compare and merge into this project.  Still waiting on any one who is part of this repo to respond

**Comment by eejag** [comment]
@yupdon Don't know if my answer is relevant or not but I'm currently using Necro 0.7.1 successfully

**Comment by yupdon** [comment]
@eejag yes it is running. As soon as my kids take a nap I'll copy the repo over and we should be a go!

**Comment by leovarmak** [comment]
@yupdon thanks man ! Waiting for the release soon

**Comment by yupdon** [comment]
Update: Got the API with protos in the project, but still have to tie the GUI to the new functions and recreate others with the new API - is ANY of the origional devs going to help or do i need to start my own repo?

**Comment by exsypher** [comment]
@yupdon novalys said in the discard channel that he will have time tomorrow to work on the bot.

**Comment by yupdon** [comment]
@exsypher Well I'm 250 errs from done. So i'll just send him the code - when done. And ya'll a temp link.

**Comment by mmalikahmed** [comment]
@yupdon Perfect bro. Time is of the essence. We don't know when evil scientists at Niantic change their API's again. So, make sure you guys up the bot so we can all make the most of it :)

**Comment by yupdon** [comment]
@Everyone eh sry guys, I got it to 43 errs (kids woke up - dev stopped until tonight). @Novalys If you want my code to help you get moving just let me know how to send it to you!

**Comment by Markmanus** [comment]
Thankyou yapdon for ur efforts.  I hope novalys will join soon.

**Comment by justintindle** [comment]
My fork has the new API integrated with the existing GUI, but pokestops appear to be missing some information and the bot appears to be soft-banned with no exp or items gained.  @Novalys @yupdon Feel free to use whatever you need to finish and pass on what I might be missing currently.

**Comment by TheOnlySeb** [comment]
I ve used NecroBot since this GUI doest work, bot works fine, setting it up is a little more complex, but the bot wiki exlains how to pretty good. Just give it a shot.

**Comment by justintindle** [comment]
Currently, the BOT in my fork is able to catch pokemon, but still having issues with the fort searching.

**Comment by justintindle** [comment]
The BOT from my fork is able to catch pokemon and search the pokestops.  There may be a few issues as I haven't tested all functionality, but the farming aspect of it appears to be working again.

**Comment by R1NZLR** [comment]
@justintindle Good work man. A couple minor adjustments, and its running beautifully.

**Comment by aaamoeder** [comment]
@justintindle @R1NZLR Sorry to ask such a noob question, but how do I integrate the updated fork from justin ? I checked the repo, but there is no new release, and I am clueless as how to merge the updated files ..

**Comment by yupdon** [comment]
@aaamoeder I'm quite certain you have to be given permission. Create your own repo. Call it based on novalys. Upload it there put out links.
